app/api/[guestId]+api.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  deleteGuestInsecure,
+  getGuestInsecure,
+  updateGuestInsecure,
+} from '../../database/guests';
+import { ExpoApiResponse } from '../../ExpoApiResponse';
+import {
+  type Guest,
+  guestsSchema,
+} from '../../migrations/00000-createTableGuests';
+
+type GuestResponseBodyGet = { guest: Guest } | { error: string };
```

Comment:
this shouldn't be on one line

follow the other format that you introduced in the other places

```
| { ... }
| { ... }
```

(for all occurrences of the 1-line union)

Reply:
Prettier is always chaning it from 
```
type GuestResponseBodyGet = 
  | { guest: Guest } 
  | { error: string };
```

to 
```
type GuestResponseBodyGet = { guest: Guest } | { error: string };
```

Reply:
multiline 
[8a8b84c](https://github.com/upleveled/expo-example-spring-2024-atvie/pull/4/commits/8a8b84c5345ddd08a18be3efc90a73f90427eade)